PORTAL/jobs/__main__.py

```diff
@@ -1050,14 +1074,18 @@ def add_cmd(name, subs=subs, *, parents=(), **kwargs):
         sub.add_argument('--clean', dest='clean', action='store_const', const=True)
         sub.add_argument('--no-push', dest='push', action='store_false')
         sub.add_argument('--push', dest='push', action='store_const', const=True)
-    sub.add_argument('reqid')
+    sub.add_argument('reqid', help=REQID_HELP)
 
     sub = add_cmd('compare', help='Compare two or more results')
     #sub.add_argument('--fmt', choices=PyperfTable.FORMATS)
     sub.add_argument('--mean-only', dest='meanonly', action='store_true')
-    sub.add_argument('--pyston', action='store_true')
-    sub.add_argument('res1')
-    sub.add_argument('others', nargs='+')
+    sub.add_argument(
+        '--pyston',
+        action='store_true',
+        help='Compare the pyston benchmarks, otherwise only compare the pyperformance benchmarks'
+    )
+    sub.add_argument('res1', help=REQID_HELP)
+    sub.add_argument('others', nargs='+', help='Request ID(s) to compare against')
```

Comment:
FWIW, these can also be upload IDs (and maybe one other thing).  The help text probably doesn't need to worry about that though.